lib/person.rb

```diff
@@ -0,0 +1,26 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(name, age, parent_permission = true)
+    @id = SecureRandom.uuid
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def is_of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age? || parent_permission
+  end
```

Comment:
- [ ] in this setup the `can_use_services?` method is also private please make sure this is a public method.